template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/validation/SafetyValveValidator.java

```diff
@@ -0,0 +1,40 @@
+package com.sequenceiq.cloudbreak.cmtemplate.validation;
+
+import java.util.List;
+
+import org.springframework.stereotype.Component;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import com.sequenceiq.cloudbreak.template.model.SafetyValve;
+
+@Component
+public class SafetyValveValidator {
+    public static final List<String> NON_XML_FORMAT_SAFETY_VALVE = List.of("hue_service_safety_valve",
+            "spark-conf/spark-defaults.conf_client_config_safety_valve",
+            "impala_cmd_args_safety_valve",
+            "hdfs_client_env_safety_valve",
+            "mapreduce_client_env_safety_valve",
+            "spark3-conf/spark-defaults.conf_client_config_safety_valve",
+            "REGIONSERVER_role_env_safety_valve");
+
+    private XmlMapper xmlMapper = new XmlMapper();
```

Comment:
Is this class thread-safe? If so, this can be made `final`. Otherwise, it should be a local variable inside `validate()`.

Reply:
This is thread safe. made it final.

Reply:
Still pending.